main.go

```diff
@@ -0,0 +1,93 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"strings"
+	// "net/url"
+	"os"
+)
+
+func main() {
+	var page int
+	page = 1
+	// params := url.Values{}
+	// params.Add("page", page)
+	for i := 0; i <155; i++{
```

Comment:
You're doing a lot of network requests here, maybe just do 3? Sorry I should have mentioned this before, but when using API's like this that we aren't paying for, try and be nice and not hit them all at once. 

For now, maybe just do three pages and use the players returned in there. 

TBH I didn't realise you were going to add pagination support, but nice.